manifests/fragment.pp

```diff
@@ -15,27 +15,19 @@
 #   anything else using this to influence the order of the content in the file
 #
 define concat::fragment(
-    $target,
-    $ensure  = undef,
-    $content = undef,
-    $source  = undef,
-    $order   = '10',
+    Variant[String, Stdlib::Compat::String]                                   $target,
+                                                                              $ensure  = undef,
+    Optional[Variant[String, Stdlib::Compat::String]]                         $content = undef,
+    Optional[Variant[String, Stdlib::Compat::String, Array]]                  $source  = undef,
```

Comment:
Okay. And you too on https://github.com/puppetlabs/puppetlabs-apt/pull/659/files#diff-f0fb8c712e70efd75993654e78e8bd59R8 then :D